controller.java

```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
         int num1 = 10;
         int num2 = 20;
 
-        // Perform addition
-        int sum = num1 + num2;
-
+int summation = math.add(number2, num1);
+        int summation = math.add(num2, num1);
```

Comment:
There is duplicate code. One line is enough.
```suggestion
        int summation = math.add(number2, num1);
```